app/models/employee.rb

```diff
@@ -1,3 +1,7 @@
 class Employee < ApplicationRecord
-has_many :reports
+  has_many :reports
```

Comment:
this suggestion is to ensure integrity data in our database, [check out this blog post](https://thoughtbot.com/blog/referential-integrity-with-foreign-keys) to know more.
```suggestion
  has_many :reports, dependent: :destroy
```